storage/src/main/scala/coop/rchain/storage/regex/Multiplier.scala

```diff
@@ -16,27 +16,35 @@ private[regex] object Multiplier {
 
   def apply(multiplier: Int): Multiplier = new Multiplier(Some(multiplier), Some(multiplier))
 
-  def apply(symbol: String): Multiplier = symbol match {
-    case "*" => presetStar
-    case "?" => presetQuestion
-    case ""  => presetOne
-    case "+" => presetPlus
+  def parse(cs: CharSequence): Option[Multiplier] = tryParse(cs).map {
```

Comment:
tryParse(cs).map(_._1)

Reply:
Updated with code that mimics RegexPattern.parse behavior (Some(x) if entire line is parsed, none otherwise)